### ISSUE TITLE
No keyboard navigation?

### ISSUE THREAD

**Issue by rdoeffinger** [Ashok-Varma/BottomNavigation#187]
**Describe the bug**
It does not seem possible to select navigation items via keyboard or TV D-Pad? 

**To Reproduce**
- Compile sample (have to comment out most of the TextFragment stuff since it does not compile)
- Start it in emulator
- Navigate using the cursor keys (or the D-Pad emulation if using a Android TV image)
- Notice that there is no way to select a different tab using the keyboard, the navigation bar does not seem to contain any focusable items at all

**Expected behavior**
It should be possible to navigate using the keyboard, which is necessary for both proper accessibility support, to improve usability on e.g. Chrombooks and to make it possible to use at all on Android TV.
